### PR TITLE
gomobile fixes

### PR DESCRIPTION
**Pull request by drwasho**
This PR contains changes for gomobile to build frameworks for iOS and Android. It is currently configured to build the framework using a Bitcoin Cash SPV wallet. 

Additional changes are required to make the selection of the wallet configurable in `\mobile\config.go`.

### CODE REVIEW

File: cmd/start.go [hunk 1]
```diff
@@ -496,6 +496,12 @@ func (x *Start) Execute(args []string) error {
 	proto.Unmarshal(dhtrec.GetValue(), e)
 
 	// Wallet
+	// Exchange rates
```
Comment (cpacia) [inline cmd/start.go]: I think this block should not be here. Exchange rates are already set in the `start`

File: repo/init.go [hunk 2]
```diff
@@ -161,7 +161,7 @@ func addConfigExtensions(repoRoot string, testnet bool) error {
 		return err
 	}
 	var w WalletConfig = WalletConfig{
-		Type:             "spvwallet",
```
Comment (cpacia) [inline repo/init.go]: We also do not want to initialize it with bitcoin cash by default here.

File: cmd/start.go (hunk 1 above)
Comment (drwasho) [reply cmd/start.go]: Ok fixed in the latest commit.

File: repo/init.go (hunk 2 above)
Comment (drwasho) [reply repo/init.go]: Fixed

**Comment by drwasho** [comment]
@cpacia changes made

**Comment by hoffmabc** [comment]
@drwasho can you please address @cpacia issues?

**Comment by drwasho** [comment]
@hoffmabc I already did, @cpacia hasn't updated his review

**Comment by placer14** [comment]
@drwasho can you help me understand what delta here that still needs to get into master? Or can we close?

**Comment by drwasho** [comment]
@placer14 the current mobile folder was never configured correctly, as Chris can attest to; the only way we ever managed to get it to work was by making changes that we part of this PR.. but no one ever got around to merging them at the time. After a while it became clear that we needed to wait for multiwallet to be finished and make similar adjustments if the mobile frameworks were failing to build or function properly.

We could close this PR and prepare a fresh new one if these changes are necessary.

**Comment by placer14** [comment]
Archiving.

**Pull request closed without merging**